org.mypaint.MyPaint.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "org.mypaint.MyPaint",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
+    "sdk": "org.gnome.Sdk",
+    "command": "mypaint",
+    "rename-desktop-file": "mypaint.desktop",
+    "rename-appdata-file": "mypaint.appdata.xml",
+    "rename-icon": "mypaint",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host"
```

Comment:
I set the entire filesystem so users don't see a limitation with the MyPaint packaged by distributions. But yes I think is better to allow the user home only. Users normally store files in their home.